plugins/modules/win_listen_ports_facts.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, DataDope (@datadope-io)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r'''
+---
+module: win_listen_ports_facts
```

Comment:
Need to specify when this module was added.

```suggestion
module: win_listen_ports_facts
version_added: '1.10.0'
```

Reply:
I've added 'version_added' key